frontend/src/components/productGrid/DropdownButton.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext, useState } from "react";
+import { getModels } from "interface/simaProInterface";
+import Select from "react-select";
+import { PrivateSectionContext } from "resources/PrivateSectionContext";
+
+const SelectModelVariable = (props) => {
+  const productID = props.productID;
+  const productName = props.productName;
+  const [selectedProducts, setSelectedProducts] = useContext(
+    PrivateSectionContext
+  );
+  const variables = getModels(productID);
+  const [selected, setSelected] = useState("Select a model");
+
+  return (
+    <Select
+      options={variables}
+      onChange={(props) => {
+        console.log(props);
+        variables.map((item) => (
+          <a
+            onClick={(props) => {
+              const newSelectedProducts = [
+                {
+                  productID: productID,
+                  productName: productName,
+                  modelID: item.modelID,
+                  modelName: item.modelName
+                }
+              ];
+              setSelected(item.modelName);
+              setSelectedProducts(newSelectedProducts);
+            }}
+            className="w3-bar-item w3-button"
+            key={item.modelID}
+          >
+            {item.modelName}
+          </a>
+        ));
+      }}
+    />
+  );
+};
+
+export default SelectModelVariable;
```

Comment:
I removed this file in the latest cleanup session since it was not used, to avoid confusion in the code @manifau

However, since we are working in git, a deleted file is not gone forever. It can be found when checking out an older state of our project. But maybe we need to discuss about that here.  

@manifau Do you want to use this dropdown? And if so, why? ☺️

Reply:
(#37 Uniform user Interface) For this task, I and Irem were deciding on a dropdown look and feel because the previous one shifts on selection. So this was another option if we do not have a solution.
And since  "a deleted file is not gone forever", it is okay :) I can retrieve it later if needed.

Reply:
Ah now it's clear to me 😃 

So you added this one, to use it if we don't get the dropdown to work properly.

Reply:
Since we got the Dropdown to work, I will remove this file again.